### PR TITLE
Support Spark Data Lake cosmos08 cluster

### PR DESCRIPTION
**Pull request by wezhang**

### CODE REVIEW

**Comment by wezhang** [comment]
retest this please

**Comment by t-rufang** [comment]
Test it now.

**Comment by wezhang** [comment]
retest this please

File: Utils/hdinsight-node-common/src/com/microsoft/azure/hdinsight/sdk/common/azure/serverless/AzureSparkServerlessClusterManager.java [hunk 1]
```diff
@@ -72,7 +72,7 @@ public static AzureSparkServerlessClusterManager getInstance() {
     private static final String REST_SEGMENT_ADL_ACCOUNT = "providers/Microsoft.DataLakeAnalytics/accounts";
 
     // FIXME!!!
-    private static final String ACCOUNT_FILTER = CommonSettings.getAdEnvironment().endpoints().getOrDefault("dataLakeSparkAccountFilter", "endswith(name, '-c09')");
+    private static final String ACCOUNT_FILTER = CommonSettings.getAdEnvironment().endpoints().getOrDefault("dataLakeSparkAccountFilter", "endswith(name, '-c09') or endswith(name, '-c08')");
```
Comment (djlan) [inline Utils/hdinsight-node-common/src/com/microsoft/azure/hdinsight/sdk/common/azure/serverless/AzureSparkServerlessClusterManager.java]: c08 and c09 is not enough

Comment (wezhang) [reply Utils/hdinsight-node-common/src/com/microsoft/azure/hdinsight/sdk/common/azure/serverless/AzureSparkServerlessClusterManager.java]: fix it.

**Pull request merged**